### PR TITLE
[Requirements] Requirement#10, Map Features From GeoJSON/Shape

### LINKED ISSUE #21: [Requirements] 10. Get All Map Features In A Shape
**Is your feature request related to a problem? Please describe.**
This issue deals with the 10th requirement from the PRD extracting all map features within a shape.

**Describe the solution you'd like**
Same requirement steps as in R09.

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by Rubix982**
# Pull Request Type

- 🏆 **Enhancements** - It introduces some changes,
  - Finishes requirement#10.1 (geojson as input) and requirement#10.2 (shape as input)
  - Vector Tiles API Adapter refactoring
  
On merge, it should close #21 

## Purpose

Changes made keeping in mind the 10th requirement

## Feedback required over

- A quick pair of :eyes: on the code
- Discussion on the technical approach 

## Mentions

- @cbeddow review, please
- @OmarMuhammedAli review, please

### CODE REVIEW

**Review by OmarMuhammedAli — APPROVED** [review]
LGTM!

**Review by cbeddow — APPROVED** [review]

File: mapillary/models/api/vector_tiles.py [hunk 1]
```diff
@@ -587,7 +593,7 @@ def __preprocess_features(
                 # Parameters accordingly
                 param="feature_type",
                 value=feature_type,
-                options=["point", "tiles"],
+                options=["point", "traffic_signs"],
```
[the remark this answers is not in the document]
Comment (Rubix982) [reply mapillary/models/api/vector_tiles.py:596]: Oops, typo.

File: mapillary/mapillary.py [hunk 2]
```diff
@@ -741,22 +738,18 @@ def get_map_features_in_shape(shape: dict, **filters: dict):
...
-    :rtype: dict
+    :return: A GeoJSON object
+    :rtype: mapillary.models.geojson.GeoJSON
 
     Usage::
         >>> import mapillary as mly
         >>> import json
         >>> mly.set_access_token('MLY|XXX')
-        >>> data = mly.images_in_shape(json.load(open('polygon.geojson', mode='r')))
+        >>> data = mly.map_features_in_shape(json.load(open('polygon.geojson', mode='r')))
```
[the remark this answers is not in the document]
Comment (Rubix982) [reply mapillary/mapillary.py:748]: @cbeddow yes, it is possible to parse a string if it is a URL.

**Comment by Rubix982** [comment]
@cbeddow required changes made.

**Comment by cbeddow** [comment]
Sounds good! merging

**Pull request merged**